src/main/java/seedu/address/storage/StorageManager.java

```diff
@@ -116,6 +120,26 @@ public void saveAppointmentBook(ReadOnlyAppointmentBook appointmentBook, Path fi
         appointmentBookStorage.saveAppointmentBook(appointmentBook, filePath);
     }
 
+
+    // ================ BudgetBook methods ==========================
+    @Override
+    public BudgetBook readBudgetBook() {
+        return this.budgetBookStorage.loadBudgetBook();
+    }
+
+    @Override
+    public void saveBudgetBook(BudgetBook budgetBook) throws IOException {
+        if (budgetBook.hasBudget()) {
+            System.out.println("There");
```

Comment:
I think you forgotten to remove the debugging print statement here as well

Reply:
Wah tks you are sharp :)))